assets.js

```diff
@@ -1,5 +1,19 @@
 export const procgenAssetsBaseUrl = `https://webaverse.github.io/procgen-assets/`;
 export const urlSpecs = {
+  simplexMap: `${procgenAssetsBaseUrl}noise/simplex-noise.png`,

```

Comment:
This was intended as the place to put GLB assets which we will sprinkle into the world, not the texture urls we will load.

To better separate it, this could be in its own constant, and this `urlSpecs` variable needs a better name to note that it is only for holding the GLB candidates for the different layers.

Reply:
Ah, I see 👍